.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+    "env": {
+        "commonjs": true,
+	"node": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    "rules": {
+        "indent": [
```

Comment:
How are you currently indenting blocks in your code? This should match that. (If you're inconsistent, pick a style, update this to reflect that, and then fix the inconsistency.)